py65/utils/console.py

```diff
@@ -25,8 +25,7 @@ def getch(stdin):
         for function signature compatibility and is ignored.
         """
         c = msvcrt.getch()
-        if isinstance(c, bytes): # Python 3
```

Comment:
The (now deleted) comment was useful to explain when this conditional code is needed.